db/migrate/20180330003212_create_items.rb

```diff
@@ -0,0 +1,10 @@
+class CreateItems < ActiveRecord::Migration[5.1]
```

Comment:
If you were working with a large amount of data, you would want to add indexes on these tables.